x-pack/legacy/plugins/ml/public/application/settings/calendars/edit/calendar_form/calendar_form.js

```diff
@@ -141,7 +167,7 @@ export const CalendarForm = ({
           options={jobIds}
           selectedOptions={selectedJobOptions}
           onChange={onJobSelection}
-          isDisabled={saving === true || canCreateCalendar === false}
+          isDisabled={saving === true || canCreateCalendar === false || isGlobalCalendar === true}
```

Comment:
Just a thought, I wonder if hiding the Jobs and Groups inputs would be better than disabling, if the 'Apply to all' switch is turned on?

![image](https://user-images.githubusercontent.com/7405507/75171894-4bc71980-5724-11ea-9bfd-d03b43bdcc67.png)

Reply:
Fixed in a7802a4837115854613b866b6e7a4b6fcb37e596